tests/ui/invalid_pyclass_enum.stderr

```diff
@@ -15,3 +15,15 @@ error: #[pyclass] can't be used on enums without any variants
    |
 16 | enum NoEmptyEnum {}
    |                  ^^
+
+error: Unit variants are not yet supported in complex enums.
+  --> tests/ui/invalid_pyclass_enum.rs:21:5
+   |
+21 |     UnitVariant,
```

Comment:
For unit variants, it would be very nice if the error message can name the conflicting variants and also hint how to make this compile:

```suggestion
error: Unit variant `UnitVariant` cannot be mixed with complex variant `StructVariant` in the same enum.
  = help: change to `UnitVariant { }` to opt in to a full variant type without any fields
  --> tests/ui/invalid_pyclass_enum.rs:21:5
   |
21 |     UnitVariant,
```

Reply:
I've added more details to these error messages, along the lines of what you suggest here, although not verbatim.